open_tms_backend/open_tms_delete_ticket_type/index.js

```diff
@@ -1,19 +1,98 @@
 import { shared } from '@appblocks/node-sdk'
 
+/**
+ * @swagger
+ * components:
+ *   securitySchemes:
+ *     bearerAuth:
+ *       type: http
+ *       scheme: bearer
+ *       bearerFormat: JWT
+ */
+
+/**
+ * @swagger
+ * /open_tms_backend/open_tms_delete_ticket_type:
+ *   post:
+ *     security:
+ *       - bearerAuth: []
+ *     summary: Delete ticket type
+ *     description: Delete ticket type.
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               ticket_type_id:
+ *                 type: string
+ *                 description: The type id
+ *                 example: dsbcndjA28
+ *     responses:
+ *       '201':
+ *         description: Deleted
+ *       '200':
+ *         description: Ok
+ *       '400':
+ *         description: Bad Request
+ *       '500':
+ *         description: Internal Server Error
+ */
+
 const handler = async (event) => {
-  const { req, res } = event
-  const { sendResponse } = await shared.getShared()
+  try {
+    const { req, res } = event
+    const { sendResponse, isEmpty, prisma, validateRequestMethod, checkHealth } = await shared.getShared();
+    const ticket_type_id = req.body.ticket_type_id;
 
-  // health check
-  if (req.params.health === 'health') {
-    return sendResponse(res, 200, { message: 'Health check success' })
-  }
+    if (checkHealth(req, res)) return;
+
+    await validateRequestMethod(req, ["POST"]);
+
+    if (isEmpty(ticket_type_id)) {
+      return sendResponse(res, 400, {
+        message: "Please provide ticket type id",
+      });
+    }
 
```

Comment:
Check if the user is role-assignee.The same logic you used for create ticket type can be used here